src/presentation/test/builder/AbstractPartnerBuilder.ts

```diff
@@ -89,6 +89,11 @@ abstract class AbstractPartnerBuilder {
     this.data.nationality2 = nationality2;
     return this;
   }
+
+  withLegalEntityRegistrationLocation(country: string) {
```

Comment:
could you change the param name from country to legalEntityRegistrationLocation, country is confusing me with the other country fields